test/dommatrix.test.js

```diff
@@ -403,8 +403,51 @@ describe('DOMMatrix', function () {
     })
   })
 
-  describe('inverse', function () {})
-  describe('invertSelf', function () {})
+  describe('invertSelf', function () {
+    it('works', function() {
+      var d = new DOMMatrix(Avals)
+      d.invertSelf()
+      assertApprox(d.m11, 0.9152542372881356)
+      assertApprox(d.m12, -0.01694915254237288)
+      assertApprox(d.m13, -0.7966101694915254)
+      assertApprox(d.m14, -0.13559322033898305)
+      assertApprox(d.m21, -1.8305084745762712)
+      assertApprox(d.m22, -0.9661016949152542)
+      assertApprox(d.m23, 1.5932203389830508)
+      assertApprox(d.m24, 1.271186440677966)
+      assertApprox(d.m31, 0.7966101694915254)
+      assertApprox(d.m32, 0.559322033898305)
+      assertApprox(d.m33, -0.711864406779661)
+      assertApprox(d.m34, -0.5254237288135594)
+      assertApprox(d.m41, 0.711864406779661)
+      assertApprox(d.m42, 0.5423728813559322)
+      assertApprox(d.m43, -0.5084745762711864)
+      assertApprox(d.m44, -0.6610169491525424)
+    })
+  })
+
+  describe('inverse', function () {
+    it('preserves the original DOMMatrix', function() {
+      var d = new DOMMatrix(Avals)
+      d.inverse()
+      assert.strictEqual(d.m11, Avals[0])
+      assert.strictEqual(d.​m12, Avals[1])
```

Comment:
This makes no sense whatsoever:
```
node-canvas\test\dommatrix.test.js:434
      assert.strictEqual(d.​m12, Avals[1])


SyntaxError: Invalid or unexpected token
    at wrapSafe (internal/modules/cjs/loader.js:1167:16)
    at Module._compile (internal/modules/cjs/loader.js:1215:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1272:10)
    at Module.load (internal/modules/cjs/loader.js:1100:32)
    at Function.Module._load (internal/modules/cjs/loader.js:962:14)
    at Module.require (internal/modules/cjs/loader.js:1140:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at C:\Users\Mike\Documents\Git\temp\node-canvas\node_modules\mocha\lib\mocha.js:250:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\Mike\Documents\Git\temp\node-canvas\node_modules\mocha\lib\mocha.js:247:14)
    at Mocha.run (C:\Users\Mike\Documents\Git\temp\node-canvas\node_modules\mocha\lib\mocha.js:576:10)
    at Object.<anonymous> (C:\Users\Mike\Documents\Git\temp\node-canvas\node_modules\mocha\bin\_mocha:637:18)
    at Module._compile (internal/modules/cjs/loader.js:1251:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1272:10)
    at Module.load (internal/modules/cjs/loader.js:1100:32)
    at Function.Module._load (internal/modules/cjs/loader.js:962:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
```

Reply:
@chearon I don't supposed you'd know how to fix this? =S

Reply:
There's a bunch of zero-width spaces in there that show up when I use Vim:

<img width="444" alt="Screen Shot 2020-08-14 at 11 19 06 AM" src="https://user-images.githubusercontent.com/78187/90265021-f8ef3480-de1f-11ea-8aff-40321e47e443.png">

Reply:
how in the world... I copied the lines from the other tests O_o

Alright, let's clean that up I guess, good catch.